pandas/core/internals/construction.py

```diff
@@ -422,6 +422,12 @@ def _get_axes(N, K, index, columns):
     return index, columns
 
 
+def _dataclasses_to_dicts(data):
```

Comment:
same here you can put the import at the top with a try/except around it

Reply:
I think that is unnecessary as this function should error if someone is calling it in the context where dataclasses are not available. The only thing a try-except add here is ambiguity about the return value (if dataclasses don't exists return empty dict?) and add an additional overhead (try/except is not a cost-free abstraction)

Reply:
k